sycl/include/sycl/ext/oneapi/matrix/matrix.hpp

```diff
@@ -24,11 +24,11 @@
 #endif // SYCL_EXT_ONEAPI_MATRIX_VERSION
 #if (SYCL_EXT_ONEAPI_MATRIX_VERSION == 2)
```

Comment:
remove matrix-jit-use. it does not work anymore because we removed SPIRV and codegen support for this.